src/py/flwr/cli/new/templates/app/pyproject.flowertune.toml.tpl

```diff
@@ -8,15 +8,15 @@ version = "1.0.0"
 description = ""
 license = "Apache-2.0"
 dependencies = [
-    "flwr[simulation]>=1.9.0,<2.0",
-    "flwr-datasets>=0.1.0,<1.0.0",
-    "hydra-core==1.3.2",
+    "flwr[simulation]>=1.10.0,<2.0",
+    "flwr-datasets>=0.3.0,<1.0.0",
```

Comment:
We thought of simplifying this in all our `pyproject.toml` since it's highly likely that these files will be revisited before `flwr 2.0` and `flwr-datasets 1.0`